subsys/net/lib/lwm2m/lwm2m_engine.c

```diff
@@ -4716,7 +4726,8 @@ int lwm2m_socket_start(struct lwm2m_ctx *client_ctx)
 	return lwm2m_socket_add(client_ctx);
 }
 
-int lwm2m_parse_peerinfo(char *url, struct sockaddr *addr, bool *use_dtls, bool is_firmware_uri)
+int lwm2m_parse_peerinfo(char *url, struct sockaddr *addr, bool *use_dtls,
+		char *desthostname,	bool is_firmware_uri)
```

Comment:
Please use spaces to separate arguments. And fixing alignment would be nice.

Reply:
Sorry, my editor do this mess.